roles/confluent.kafka_broker/tasks/set_principal.yml

```diff
@@ -33,6 +33,28 @@
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['sasl_protocol'] | default(sasl_protocol) | normalize_sasl_protocol == 'none'
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['ssl_enabled'] | default(ssl_enabled) | bool
     - kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['ssl_mutual_auth_enabled'] | default(ssl_mutual_auth_enabled) | bool
+    - not fips_enabled|bool
+  no_log: "{{ mask_sensitive_logs|bool }}"
+
+- name: Extract Distinguished Name from BCFKS Keystore - SSL Mutual Auth
```

Comment:
is there a molecule test to validate this?

Reply:
Nope. No rbac with fips scenario coverage. Will add and test with that.

Reply:
Can't get this combo to work. Searching through slack history, I see that we haven't got fips to work with mtls. And rbac without mtls won't touch this piece of code. 
So, no molecule test to validate this. Tested locally for both kind of certificates.